types/algoliasearch-helper/index.d.ts

```diff
@@ -478,6 +478,7 @@ declare namespace algoliasearchHelper {
     getHierarchicalRefinement(facetName: string): string[];
     getNumericRefinements(facetName: string): SearchParameters.OperatorList[];
     getNumericRefinement(attribute: string, operator: SearchParameters.Operator): Array<number | number[]>;
+    getQueryParams(): any;
```

Comment:
```suggestion
    getQueryParams(): SearchParameters;
```

Reply:
@Haroenv Would it make sense to also update helper.searchOnce to accept type QueryParameters?

Reply:
@Kamshak, yes, that's correct!